doc-content/enterprise-only/employee-rostering/wb-employee-rostering-example-deploying-in-central-proc.adoc

```diff
@@ -6,7 +6,7 @@
 .Prerequisites
 
 * {PRODUCT} has been downloaded and installed. For installation options, see {URL_INSTALLING_AND_CONFIGURING}#assembly-planning[_{PLANNING_INSTALL}_].
-* You have started the {KIE_SERVER} and logged in to {CENTRAL} with a user that has `admin` permissions. 
+* You have started {PRODUCT} as described the installation documentation that you used and you are logged in to {CENTRAL} with a user that has `admin` permissions.
```

Comment:
```suggestion
* You have started {PRODUCT}, as described in the installation documentation, and you are logged in to {CENTRAL} as a user with `admin` permissions.
```
No real sins here, except that I had to add an "in" after "described". All my other suggestions are for my own idea of clarity.

Reply:
Thanks. That is better.